discovery/src/test/java/com/ibm/watson/developer_cloud/discovery/v1/DiscoveryServiceIT.java

```diff
@@ -101,6 +101,7 @@
  */
 public class DiscoveryServiceIT extends WatsonServiceTest {
   private static final String DISCOVERY_TEST_CONFIG_FILE = "src/test/resources/discovery/test-config.json";
+  private static final String DISCOVERY1_TEST_CONFIG_FILE = "src/test/resources/discovery/issue517.json";
```

Comment:
i would probably just update the test-config to have a `json_normalizations` attribute.

basically make a copy of this test:

```
@Test
   public void getConfigurationIsSuccessful() {
     CreateConfigurationResponse createResponse = createTestConfig();
 
     GetConfigurationRequest getRequest = new GetConfigurationRequest.Builder(environmentId,
         createResponse.getConfigurationId()).build();
     GetConfigurationResponse getResponse = discovery.getConfiguration(getRequest).execute();
 
     assertEquals(createResponse.getName(), getResponse.getName());
   }
```

and do the same validation you have now:

```
assertEquals(1, getResponse.getConversions().getJson().size());
```

maybe you could call it
```
public void getConfigurationHasJsonNormalization() { ... }
```

but if it is easier to manage using issue numbers instead of this then go for it